modules/installation-gcp-permissions.adoc

```diff
@@ -35,11 +35,19 @@ When you attach the `Owner` role to the service account that you create, you gra
 .Required roles for using the Cloud Credential Operator in passthrough mode
 * Compute Load Balancer Admin
 
+.Required roles for using short-lived credentials
+* IAM Service Account User
```

Comment:
The "Required roles for the installation program" has listed "Service Account User" role already. Besides, please also take a look at [my comment](https://issues.redhat.com/browse/OCPBUGS-38723?focusedId=25567899&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-25567899), thanks!